### ISSUE TITLE
Very rare bottomless cave

### ISSUE THREAD

**Issue by florench** [doublespeakgames/adarkroom#516]
I read in the [Events section](https://github.com/doublespeakgames/adarkroom/wiki/Suggestions#more-events) of the Suggestions page on the wiki: "I love the idea of ultra-rare things that only a handful of people will ever see."

How about there is a 1 in 10,000 chance that when you continue deep into a cave, you fall into a bottomless pit and die?

> You step and fall... and fall...  You have time to wish you'd turned back.

**Comment by tehp** [comment]
This could be very cool. +1

**Comment by ChrisMcLain** [comment]
Might want to keep in mind that the [design guidelines](https://github.com/doublespeakgames/adarkroom/wiki/Design-Guide#narrative) say that personal pronouns shouldn't be used. It also states the following, which means this idea might go against the event design principles:

> Generally, "bad stuff" only happens when the player is lingering in the forest or village.

 I like the idea, though.

**Comment by ddugovic** [comment]
More appropriate according to that principle would be the possibility of dying, only to awaken and discover that it was only a premonition.

**Comment by florench** [comment]
Yes, not using personal pronouns makes my suggestion violate the design guidelines.  You can "die" during a quest though, can't you? I have.

**Comment by gitupofftheflooranddosomework** [comment]
Do we think that we want this or can we close it ?

**Comment by esaruoho** [comment]
this feels like it would just make an already challenging game even more challenging, to the point of being unfriendly. it's a way to discourage people from exploring caves. i would tend to say no we don't want this.
